tests/test_reordering.f90

```diff
@@ -75,7 +75,7 @@ program test_reorder
   L_global = [2*pi, 2*pi, 2*pi]
 
   ! Domain decomposition in each direction
-  nproc_dir = [nproc, 1, 1]
+  nproc_dir = [1, 1, nproc]
```

Comment:
As for `test_omp_transeq.f90`

Reply:
Yes, this is because 2decomp&fft only allows 2D decomposition and as it is implemented at the moment it is in y and z.